pkg/configuration/digger_config.go

```diff
@@ -388,6 +407,41 @@ func (c *DiggerConfig) GetWorkflowConfiguration(projectName string) WorkflowConf
 	return *workflow.Configuration
 }
 
+func checkSteps(steps []Step) error {
+	for _, step := range steps {
+		err := step.IsEmptyValueAndExtraArgs()
+
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (c *DiggerConfig) CheckWorkflowConfiguration() error {
+	for _, project := range c.Projects {
+		workflows := c.Workflows
+		workflow, ok := workflows[project.Workflow]
+
+		if !ok {
+			return nil
```

Comment:
here we should raise an error of "project specifies workflow not in list of workflows